src/components/PhotoFrame.tsx

```diff
@@ -517,48 +564,37 @@ const PhotoFrame = ({
         if (!fetching[item.dataIndex]) {
             try {
                 fetching[item.dataIndex] = true;
-                let url: string;
+                let URLs: string[];
```

Comment:
so acronyms are capitalised only when they are not the first word. 😅  

so it should be written `urls` 😅  and capitalised,

because first letter capital is used to signify that its some sort of constant.

Reply:
Oh, didn't knew this.